### DISCUSSION TITLE
Can we please create more categories before this becomes messy?

### DISCUSSION THREAD

**Discussion by TerryFrench** [meshtastic/Discussions#132] [North America]
I propose 
Canada 
USA 
Mexico (unless it’s Central America?)
Under each country, the province or state. 
example BC for British Columbia under Canada
WA for Washington state under USA

and within the states / provinces, whatever people want 

example
North America/USA/CA/Bay Area Network…

if needed I volunteer to go create the categories but I would need to be granted that category permission. 
Thanks

**Comment by garthvh** [comment]
There are 5 total discussions so far, so probably not time to get more granular yet.

**Reply by TerryFrench** [reply]
I somewhat disagree as while there are almost no discussions yet, things will get messy fast :-) 
But I'm merely a guest here and I indeed wonder where the discourse discussions went. Maybe on Discord?
